ballista/scheduler/src/state/mod.rs

```diff
@@ -248,7 +262,7 @@ impl<T: 'static + AsLogicalPlan, U: 'static + AsExecutionPlan> SchedulerState<T,
     async fn launch_tasks(
```

Comment:
I wonder if it makes sense to propagate `sender:EventSender<QueryStageSchedulerEvent>,` and `task.manager.launch_multi_task` and cancel the jobs there?

`task.manager.launch_multi_task` semantics is not clean, do yield error if only one task fail?